libbeat/processors/decode_xml_wineventlog/docs/decode_xml_wineventlog.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[decode-xml-wineventlog]]
+=== Decode XML Wineventlog
+
+++++
+<titleabbrev>decode_xml_wineventlog</titleabbrev>
+++++
+
+experimental[]
+
+The `decode_xml_wineventlog` processor decodes Windows Event Log data in XML format that is stored under the `field`
+key. It outputs the result into the `target_field`.
+
+The output fields will be the same as the
+{winlogbeat-ref}/exported-fields-winlog.html#_winlog[winlogbeat winlog fields].
+
+The supported configuration options are:
+
+`field`:: (Required) Source field containing the XML. Defaults to `message`.
+
+`target_field`:: (Required) The field under which the decoded XML will be
+written. To merge the decoded XML fields into the root of the event specify
+`target_field` with an empty string (`target_field: ""`). The default value is
+`winlog`.
+
+`overwrite_keys`:: (Optional) A boolean that specifies whether keys that already
+exist in the event are overwritten by keys from the decoded XML object. The
+default value is `true`.
+
+`map_ecs_fields`:: (Optional) A boolean that specifies whether to map additional ECS
+fields when possible. The default value is `true`.
+
+`document_id`:: (Optional) XML key to use as the document ID. If configured, the
```

Comment:
I don't think this will be useful for event log XML data. I can't think of a field that has enough uniqueness on its own. I'd would probably use a fingerprint processor to combine a few fields like record_id, channel, computer_name, timestamp if I wanted a unique `_id`.